api/node/routes.go

```diff
@@ -20,9 +20,17 @@ type FacadeHandler interface {
 	TpsBenchmark() *statistics.TpsBenchmark
 	StatusMetrics() external.StatusMetricsHandler
 	GetQueryHandler(name string) (debug.QueryHandler, error)
+	GetPeerInfo(pid string) ([]interface{}, error)
 	IsInterfaceNil() bool
 }
 
+// TODO remove this struct, use shared.GenericAPIResponse
```

Comment:
done, EN-6756

Reply:
why? it's just a merge which I have to do anyway

Reply:
not quite, you will merge it into development, this will be merged in release-candidate which will go into master and then (through a merge that will be done automatically as there won't be conflicts) into dev. If this merge will be done *after* you merged into dev the generic api response, will miss working on TODOs. Task can be safely closed afterwords if you make the changes but we won't forget to do it.